03-postgres-typeorm/src/app.module.ts

```diff
@@ -0,0 +1,28 @@
+import { Module } from '@nestjs/common';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
+import { MongooseModule } from '@nestjs/mongoose';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { AuthModule } from './auth/auth.module';
+import { UserModule } from './user/user.module';
+import { PostgreSqlDataSource } from './config/ormConfig';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { EmployerModule } from './employer/employer.module';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot({
+      isGlobal: true,
+      envFilePath: `.env`,
+    }),
+    TypeOrmModule.forRoot(PostgreSqlDataSource),
```

Comment:
rather than injecting orm here, it would be great if we create a separate module for it which also contain the migration config and transaction service too